package.json

```diff
@@ -30,14 +30,14 @@
     "lodash.set": "^4.3.2"
   },
   "peerDependencies": {
-    "graphql": "^14.0.0"
+    "graphql": "^14.0.0 || ^15.0.0"
```

Comment:
please revert this as well - it's clear from the failing unit test that if this package is used with graphql 15 (provided as a peer dependency by consumers) that something breaks. We would likely need to make some changes and fully understand what has changed between graphql 14 and 15 in order to confidently put graphql 15 as a peerDependency.

Reply:
I have reverted back to graphql 14 for both the dev and peer dependency